Documentation/direct-tools.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Direct Tools
+weight: 74
+indent: true
+---
+
+# Direct Tools
+
+Rook is designed with Kubernetes design principles from the ground up. This topic is going to escape the bounds of Kubernetes storage and show you how to 
+use block and file storage directly from a pod without any of the Kubernetes magic. The purpose of this topic is to help you quickly test a new configuration,
+although it is not meant to be used in production. All of the benefits of Kubernetes storage including failover, detach, and attach will not be available.
+If your pod dies, your mount will die with it.
+
+## Block Storage Tools
+
+After you have created a pool as described in the [Block Storage](block.md) topic, you can create a block image and mount it directly in a pod.
+This example will show how the Ceph rbd volume can be mounted in the toolbox pod.
+
+After you have started and connected to the [Rook toolbox](toolbox.md), proceed with the following commands in the toolbox.
+
+Create a volume image (10MB):
+```bash
+rbd create replicapool/test --size 10
+rbd info replicapool/test
+
+# Disable the rbd features that are not in the kernel module
+rbd feature disable replicapool/test fast-diff deep-flatten object-map
+```
+
+Map the block volume and format it and mount it:
+```bash
+# Map the rbd device. If this command appears to hang, ctrl-c and check if it actually completed successfully by continuing.
+rbd map replicapool/test
+
+# Find the device name, such as rbd0
+lsblk | grep rbd
+
+# Format the volume (only do this the first time or you will lose data)
+mkfs.ext4 -m0 /dev/rbd0
+
+# Mount the block device
+mkdir /tmp/rook-volume
+mount /dev/rbd0 /tmp/rook-volume
+```
+
+Write and read a file:
+```bash
+echo "Hello Rook!" > /tmp/rook-volume/hello
+cat /tmp/rook-volume/hello
+```
+
+### Unmount the Block device
+
+Unmount the volume and unmap the kernel device:
+```bash
+umount /tmp/rook-volume
+rbd unmap /dev/rbd0
+```
+
+## Shared Filesystem Tools
```

Comment:
i'm wondering if this topic should be oriented to creating a temp/test filesystem from scratch, and to not have a dependency on having run through the kube-registry example.  is that possible or does that not really make sense because it would require deploying MDS instances anyways?

Reply:
Right, you would have to start mds. My philosophy for this topic was "ok, you set up your block or file storage with Rook. Now try to consume it to quickly test it is working"